ush/config.sh_rrfs_a_enkf_c3

```diff
@@ -159,19 +162,24 @@ if [[ ${DO_ENSEMBLE}  == "TRUE" ]]; then
    NUM_ENS_MEMBERS_FCST=9
    if [[ ${DO_ENSFCST} == "TRUE" ]] ; then
      NUM_ENS_MEMBERS=${NUM_ENS_MEMBERS_FCST}
-#     DO_SPPT=TRUE
-     DO_SPP=TRUE
-#     SPPT_MAG=0.5
      WTIME_RUN_FCST="04:45:00"
      WTIME_MAKE_LBCS="01:30:00"
      TAG="cefcst38"
 
      LAYOUT_X="31"
      LAYOUT_Y="32"
      NNODES_RUN_FCST="16"
+
+     DO_SPP="TRUE"
+     DO_SPPT="TRUE"
+     DO_SKEB="TRUE"
+     SPPT_MAG="0.6"
```

Comment:
Are all the stochastic physics turned on in the real-time run?

Reply:
It is tested on v0.4.0. EMC has good results using them but we need to talk with them to make decision.